service/controller/v1/resource/chart/create.go

```diff
@@ -36,7 +37,15 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 			}
 		}()
 
-		err = r.helmClient.InstallFromTarball(tarballPath, ns)
+		// we need to pass the ValueOverrides option to make the install process
+		// use the default values and prevent error on nested values:
+		// {rpc error: code = Unknown desc = render error in "cnr-server-chart/templates/deployment.yaml":
+		// template: cnr-server-chart/templates/deployment.yaml:20:26: executing "cnr-server-chart/templates/deployment.yaml"
+		// at <.Values.image.reposi...>: can't evaluate field repository in type interface {}}
```

Comment:
I would like a bit more format/structure to help my weak human eyes parsing this. 

```
// We need to pass the ValueOverrides option to make the install process
// use the default values and prevent errors on nested values.
//
//     {
//     	 rpc error: code = Unknown desc = render error in "cnr-server-chart/templates/deployment.yaml":
//     	 template: cnr-server-chart/templates/deployment.yaml:20:26:
//     	 executing "cnr-server-chart/templates/deployment.yaml" at <.Values.image.reposi...>: can't evaluate field repository in type interface {}
//     }
//

```

Reply:
Of course, thx done.